docs/source/ReferenceGuide.rst

```diff
@@ -24,6 +27,104 @@ results location::
 
   JOB LOG    : $HOME/avocado/job-results/job-2015-06-10T10.44-49ec339/job.log
 
+
+Test References
+---------------
+
+A Test Reference is a string that can be resolved into
+(interpreted as) one or more tests by the Avocado Test Resolver.
+Given resolver plugins are free to interpret a test reference,
```

Comment:
It was more a *suggestion* on how the sentence sounds... "Given" + plural sounds a little strange to me... but it may be correct, I'm not sure.